bin/metric-postgres-locks.rb

```diff
@@ -79,33 +82,39 @@ class PostgresStatsDBMetrics < Sensu::Plugin::Metric::CLI::Graphite
          default: nil
 
   include Pgpass
+  include Pgdatabases
 
   def run
     timestamp = Time.now.to_i
 
     locks_per_type = Hash.new(0)
     pgpass
+    databases = pgdatabases
+    # connect only to first database and get information for all databases through SQL queries
     con = PG.connect(host: config[:hostname],
-                     dbname: config[:database],
+                     dbname: databases.first,
```

Comment:
I think this still needs to be updated right?

Reply:
Updated in which way? It connects to the first DB and then queries the information for all databases. There is no need to connect to each DB individually.